Core/clim-basic/drawing/medium.lisp

```diff
@@ -96,7 +96,8 @@
       ((:sans-serif) 3)))
 
   (defun face-key (face)
-    (if (equal face '(:bold :italic))
+    (if (or (equal face '(:bold :italic))
+            (equal face '(:italic :bold)))
```

Comment:
I suggest checking the text style parameters and constructing the key at the same time (and adding a test): https://github.com/scymtym/McCLIM/commit/35398b64119088dce75715a019e1385b179e0c37

Reply:
I've cherry-picked your change and split it in two (change to internals and a test).